packages/client-utils/test/snapshots/exports.test.js.md

```diff
@@ -0,0 +1,33 @@
+# Snapshot report for `test/exports.test.js`
+
+The actual snapshot is saved in `exports.test.js.snap`.
+
+Generated by [AVA](https://avajs.dev).
+
+## index
+
+> Snapshot 1
+
+    [
+      'Natural',
+      'asBoardRemote',
```

Comment:
**suggestion**: how about deprecating `asBoardRemote` and `makeFromBoard`?

Recent work seems to treat the whole `BoardRemote` thing as a nuisance:

```console
$ ack XXX | grep -i boardremote
boot/test/orchestration/contract-upgrade.test.ts:65:      // @ts-expect-error XXX BoardRemote
boot/test/orchestration/restart-contracts.test.ts:51:      // @ts-expect-error XXX BoardRemote
boot/test/bootstrapTests/lca.test.ts:60:        // @ts-expect-error XXX BoardRemote
boot/test/bootstrapTests/lca.test.ts:88:        // @ts-expect-error XXX BoardRemote
boot/test/bootstrapTests/lca.test.ts:111:          // @ts-expect-error XXX BoardRemote
```